css/_components.team-photos.css

```diff
@@ -0,0 +1,166 @@
+/* Classes for team photos negative top margin BEGIN */
+.u-overflow-content {
+  @media (--small-ish) { overflow: visible; }
+}
+
+.u-mr-overflow-content {
+  @media (--small-ish) { margin-bottom: 8rem; }
+  @media (--medium-ish) { margin-bottom: 14rem; }
+}
+/* Classes for team photos negative top margin END */
+
```

Comment:
This line can also be removed. On top of this, I believe since this is a new different block than the file name suggests, for the simplicity purposes we could extract `u-overflow-content` to a new file called `_utility.overflow-content.css`.

Also, judging by the DOM structure when you're using the two selectors above, `.u-mr-overflow-content` should become a modifier of `.u-overflow-content`: `.u-overflow-content--mr`.